engine/runtime-language-epb/src/main/java/org/enso/interpreter/epb/HostClassLoader.java

```diff
@@ -201,7 +202,8 @@ Object readMember(String member) throws UnknownIdentifierException {
       var ctx = EpbContext.get(null);
       return ctx.getEnv().asHostSymbol(clazz);
     } catch (ClassNotFoundException ex) {
-      throw UnknownIdentifierException.create(member, ex);
```

Comment:
As the [javadoc](https://www.graalvm.org/truffle/javadoc/com/oracle/truffle/api/interop/UnknownIdentifierException.html#create(java.lang.String,java.lang.Throwable)) states, the throwable must extends `AbstractTruffleException`, otherwise the `create` method just fails with `IllegalArgumentException`. And that is what was happening. I have removed the throwable parameter and added a log statement instead.